### PR TITLE
WebXR update

### LINKED ISSUE #519: Exit fullscreen button doesn't always exit fullscreen
We received a report from a user that when they manually caused `<model-viewer>` to go fullscreen (by invoking the `requestFullscreen()` method on it directly), the exit fullscreen button introduced in #501 appeared but did not seem to do anything.

### LINKED ISSUE #1098: Fullscreen AR fallback is currently broken in Chrome

### PR DESCRIPTION
**Pull request by elalish**
Fixes #519 
Fixes #1098 

This adds the X exit button to the top-left of webXR; it takes the place of the exit fullscreen button as I've also removed the fullscreen fallback, which was broken anyway. It also switches webxr to the default ar experience. There is also some cleanup of vestigial codepaths and more complete post-session state reset.

A breaking change (though it was never documented) is that the `exit-fullscreen-button` slot is now the `exit-webxr-button` slot.

### CODE REVIEW

File: packages/model-viewer/src/features/ar.ts [hunk 1]
```diff
@@ -98,12 +98,12 @@ export type QuickLookBrowser = 'safari'|'chrome';
 const deserializeQuickLookBrowsers =
     enumerationDeserializer<QuickLookBrowser>(['safari', 'chrome']);
 
-export type ARMode = 'quick-look'|'scene-viewer'|'webxr'|'fallback'|'none';
+export type ARMode = 'quick-look'|'scene-viewer'|'webxr'|'none';
 
 const deserializeARModes = enumerationDeserializer<ARMode>(
-    ['quick-look', 'scene-viewer', 'webxr', 'fallback', 'none']);
```
Comment (klausw) [inline packages/model-viewer/src/features/ar.ts:104]: Drive-by question: if I'm understanding it right, this is removing 'fallback' from the list of recognized values. If a site is still using this in an attribute, will it be ignored silently, or does it cause an error?

File: packages/model-viewer/src/template.ts [hunk 2]
```diff
@@ -344,6 +340,17 @@ canvas.show {
 
   <div class="slot default">
     <slot></slot>
+    
+    <div class="slot exit-webxr-button">
```
Comment (cdata) [inline packages/model-viewer/src/template.ts:344]: Just my two cents, but I would give this a name that describes its semantic purpose. WebXR is a big umbrella, and doesn't necessarily capture what the user is trying to do when configuring the slot (it just adds ambiguity to the purpose of the button).

File: packages/model-viewer/src/features/ar.ts (hunk 1 above)
Comment (cdata) [reply packages/model-viewer/src/features/ar.ts:104]: It will be ignored silently

File: packages/model-viewer/src/template.ts (hunk 2 above)
Comment (elalish) [reply packages/model-viewer/src/template.ts:344]: Can you be more specific (or recommend a name)? `exit-webxr-button` seems pretty precise to me, but I'm probably just looking at it differently.

Comment (cdata) [reply packages/model-viewer/src/template.ts:344]: I think this button is specifically used for exiting AR. If you were to support VR some day, you might do it via WebXR, but I suspect that this slot wouldn't apply (or at least should be interpreted differently). Maybe I'm wrong.

Just speaking generally, WebXR is a body of API around which many, various UX can be realized (not necessarily AR-specific UX).

**Pull request merged**